### PR TITLE
001 through 004 and 006 selenium tests

### PR DESCRIPTION
**Pull request by sebastianczajkowski**
Add selenium tests for:
US001
US002
US003
US004
US006

### CODE REVIEW

File: selenium/jobRole.test.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
+const webdriver = require('selenium-webdriver');
+const chrome = require('selenium-webdriver/chrome');
+const chromedriver = require('chromedriver');
+const { fail } = require('yargs');
```
Comment (TheoHodges) [inline selenium/jobRole.test.js:4]: Is line 4 definitely supposed to be here? Looks like something VSCode often unhelpfully adds in for me. If it's okay then I'll approve

**Review by michaltangri — CHANGES_REQUESTED** [review]
Shouldn't the webdriver be build once in a test suite?

File: selenium/jobRole.test.js (hunk 1 above)
Comment (sebastianczajkowski) [reply selenium/jobRole.test.js:4]: It's not :D I've removed it just now

**Comment by michaltangri** [comment]
Nevermind, I didn't think about the fact that if you close the driver, it won't be available in other tests.

**Review by michaltangri — APPROVED** [review]

File: selenium/jobRole.test.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
+const webdriver = require('selenium-webdriver');
+const chrome = require('selenium-webdriver/chrome');
+const chromedriver = require('chromedriver');
+
+chrome.setDefaultService(new chrome.ServiceBuilder(chromedriver.path).build());
+
+describe("Test the job role table", () => {
+   test("The header above the table should be Job Roles", async () => {
+      const driver = new webdriver.Builder()
```
Comment (TheoHodges) [inline selenium/jobRole.test.js:9]: Could you refactor out lines 9-11 because they are repeated in all tests?

**Review by TheoHodges — CHANGES_REQUESTED** [review]

File: selenium/jobRole.test.js (hunk 2 above)
Comment (sebastianczajkowski) [reply selenium/jobRole.test.js:9]: I'm on it

**Review by TheoHodges — APPROVED** [review]

**Pull request merged**